Sources/Keystone/Keystone_Prefix.pch

```diff
@@ -0,0 +1,15 @@
+//
+// Prefix header for all source files of the 'WordPress' target in the 'WordPress' project
+//
+
+#ifdef __OBJC__
+	// 3rd Party    
+    @import CocoaLumberjack;
```

Comment:
This is only used and needed for Objective-C code. It's a simplified version of pch.